coremltools/converters/tensorflow/test/test_tf_2x.py

```diff
@@ -11,6 +11,79 @@
 import pytest
 
 
+@unittest.skipUnless(HAS_TF_2, 'missing TensorFlow 2+.')
+class TestSingleOp(unittest.TestCase):
+    # In this class we test tensorflow 2.x op without using Keras API
+
+    def setUp(self):
+        self.saved_model_dir = tempfile.mkdtemp()
+
+    def _test_coreml(self, model, input_dic=None, output_names=None):
+
+        # Get concrete function
+        tf.saved_model.save(model, self.saved_model_dir)
```

Comment:
We might be able to get a concrete function from the model (eager function) without `save()` and `load()`.

Reply:
Good! Let me check this.

Reply:
Done

Reply:
Awesome!